include/uapi/linux/bcm2835-isp.h

```diff
@@ -0,0 +1,317 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Is that the right licence for userspace files? Doesn't it impose GPLv2 on the userspace app then?

videodev2.h is /* SPDX-License-Identifier: ((GPL-2.0+ WITH Linux-syscall-note) OR BSD-3-Clause) */

Reply:
Done.  I've followed videodev2.h here.